cleanlab/rank.py

```diff
@@ -57,109 +118,328 @@ def order_label_issues(
       The columns must be ordered such that these probabilities correspond to class 0,1,2,...
       `pred_probs` should have been computed using 3 (or higher) fold cross-validation.
 
-    rank_by : str ['normalized_margin', 'self_confidence']
-      Method to order label error indices (instead of a bool mask), either:
-        'normalized_margin' := normalized margin (p(label = k) - max(p(label != k)))
-        'self_confidence' := [pred_probs[i][labels[i]] for i in label_issues_idx]
+    method : {"self_confidence", "normalized_margin", "confidence_weighted_entropy"}, default="normalized_margin"
+      Label quality scoring method. Default is "normalized_margin".
```

Comment:
[Nit: this is unnecessary] 
Default is "normalized_margin".

Reply:
Agree, I removed it. Thanks.